src/App.js

```diff
@@ -3,20 +3,24 @@ import './App.css';
 import { BrowserRouter as Router, Route, Routes } from 'react-router-dom';
 import React, { useState } from 'react';
 import Login from './components/Login';
+import Users from './components/Users';
 import WorkSpace from './components/WorkSpace';
 import theme from "./themes";
 import { ThemeProvider } from '@mui/material';
 
 function App() {
   return (
-    <ThemeProvider theme={theme}>
-      <Router>
-        <Routes>
-          <Route path="/" element={<Login />} />
-          <Route path="/workspace" element={<WorkSpace />} />
-        </Routes>
-      </Router>
-    </ThemeProvider>
+    <div className="App">
+      <ThemeProvider theme={theme}>
+        <Router>
+          <Routes>
+          {/* <Route path="/" element={<Login />} /> */}
+          <Route path="/" element={<Users />} />
```

Comment:
Login component should be implemented in Login component. Because we want to reuse the Users component in the SideNavBar